info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -261,18 +274,56 @@ void storeElectionContestInDatabase(String electionQueryId, JsonObject contest)
     datastore.update(electionEntity);
   }
 
-  // @TODO [Find incumbency.]
+  /**
+   * Queries the API for the representatives by division JSON. then returns all representatives in
+   * a map that can be checked to verify incumbency.
+   */
+  private Map<String, List<String>> getIncumbents(String division) {
+    try {
+      String queryUrl = String.format("%s&ocdDivisionId=%s", REPRENTATIVE_QUERY_URL, division);
+      JsonObject representatives = queryCivicInformation(queryUrl);
+      return getIncumbents(representatives);
+    } catch (IOException e) {
+        System.out.println(
+          String.format(
+              "[ERROR] Failed to query the Civic Information API for %s: %s.", "representatives", e));
+      return new HashMap<String, List<String>>();
+    }
+  }
+
+  Map<String, List<String>> getIncumbents(JsonObject representatives) {
+    Map<String, List<String>> mapOfIncumbents = new HashMap<>();  
+    JsonArray offices = representatives.getAsJsonArray("offices");
+    JsonArray officials = representatives.getAsJsonArray("officials");
+    for (JsonElement eachOffice : offices) {
+      JsonObject office = (JsonObject) eachOffice;
+      String officeName = office.get("name").getAsString();
+      JsonArray officialIndicesArray =  office.getAsJsonArray("officialIndices");
+      List<String> incumbents = new ArrayList<>();
+      for (JsonElement index : officialIndicesArray) {
+        JsonObject official = (JsonObject) officials.get(index.getAsInt());
+        incumbents.add(official.get("name").getAsString());
+      }
+      mapOfIncumbents.put(officeName, incumbents);
+    }
+    return mapOfIncumbents;
+  }
+
   /**
    * Stores information of a {@code candidate} in the database, and updates the candidate's running
    * position's information for the election. Information includes: name, party affiliation,
    * incumbency status, and news articles related to the candidate.
    */
   private void storeElectionContestCandidateInDatabase(JsonObject candidate,
-      List<Value<String>> candidateIds, List<Value<Boolean>> candidateIncumbency) {
+      List<Value<String>> candidateIds, List<Value<Boolean>> candidateIncumbency, 
+      List<Value<String>> candidatePositions, Value<String> position, Map<String, List<String>> incumbents) {
     String name = candidate.get("name").getAsString();
     String party = candidate.get("party").getAsString();
-    // @TODO [May expand to other information to uniquely identify a candidate. Currently,
-    // candidate information includes only name and party affiliation.]
+    String email = getField(candidate, "email");
+    String phone = getField(candidate, "phone");
+    String photoUrl = getField(candidate, "PhotoUrl");
```

Comment:
Didn't kathy mention that this was photoUrl?

Reply:
I think she changed it. When merging to master It will be fixed